org.lflang/src/org/lflang/generator/rust/CargoDependencySpec.java

```diff
@@ -28,6 +28,7 @@
 import java.util.LinkedHashMap;
```

Comment:
Why do these Rust files change? Pinging @oowekyala to take a look at the changes

Reply:
I don't know about this either, but I did see @oowekyala chime in recently. Are we good to go with this, @oowekyala?